src/types/course.ts

```diff
@@ -4,10 +4,30 @@ export interface ICourseItem {
   thumbnail: string;
   region: string;
   period: string;
-  theme: string[];
+  themes: string[];
   places: string[];
   likes: number;
   isBookmarked: boolean;
   nickname: string;
   profileUrl: string;
 }
+
+export interface CourseFilter {
+  placeId?: number;
+  period?: string;
+  keyword?: string;
+  region?: string;
+  spot?: string[];
+  theme?: string[];
+  sorting?: string;
+  page?: number;
+  size?: number;
+  sort?: string;
+}
+
+export const sortOrder = {
+  CREATED_AT: 'createdAt',
+  DESC: 'desc'
+} as const;

```

Comment:
```suggestion
export const sortOrder = {
  CREATED_AT: '인기순',
  DESC: '최신순'
} as const;
```
request가 변경돼서 string값 바꿔주셔야됩니다 ☺️☺️

Reply:
넵 ㅎㅎ 수정후 머지하겠습니다!